cvat/settings/base.py

```diff
@@ -40,6 +40,7 @@
 
 ALLOWED_HOSTS = os.environ.get('ALLOWED_HOSTS', 'localhost,127.0.0.1').split(',')
 INTERNAL_IPS = ['127.0.0.1']
+SECRET_KEY = os.environ.get("DJANGO_APP_SECRET", "")
```

Comment:
```suggestion
SECRET_KEY = os.environ.get("DJANGO_SECRET_KEY", "")
```

Let's use the same name.